src/components/TodoInsert.js

```diff
@@ -0,0 +1,34 @@
+import './TodoInsert.css';
+import { BsPlusCircle } from 'react-icons/bs';
+import { useState } from 'react';
+
+const TodoInsert = ({ onInsertTodo }) => {
+  const [value, setValue] = useState('');
+
+  const onChange = (e) => {
```

Comment:
이벤트 핸들러는 보통 `handle + 대상 + 동작` 형식으로 짓습니다.

```suggestion
  const handleInputChange = (e) => {
```

Reply:
이 부분 처음이라 제가 간과했네요 😅
다음부터는 네이밍룰을 지키도록 하겠습니다!